client/src/components/Info.style.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
```

Comment:
You don't need this here since you use ```flex-flow``` below, which also defines the flex-direction

Reply:
like Oliver said, you can either remove ```flex-direction``` or ```flex-flow``` as they are the same thing!!